core/src/UI/AuthorizationResult.cs

```diff
@@ -87,6 +87,13 @@ internal AuthorizationResult(AuthorizationStatus status)
 
         public void ParseAuthorizeResponse(string webAuthenticationResult)
         {
+            if (!string.IsNullOrWhiteSpace(webAuthenticationResult) &&
+                webAuthenticationResult.StartsWith(Constants.FormPostPrefix, StringComparison.OrdinalIgnoreCase))
+            {
+                this.Code = webAuthenticationResult.Substring(Constants.FormPostPrefix.Length);
+                return;
+            }
```

Comment:
should it be considered to refactor this a bit more ensuring you get other values set? e.g. state, error?

Reply:
Yes, refactored this part to return a url instead.